tensorflow/core/kernels/conv_grad_input_ops.cc

```diff
@@ -48,7 +49,7 @@ limitations under the License.
 #include "tensorflow/core/platform/stream_executor.h"
 #endif  // GOOGLE_CUDA
 
-#ifdef TENSORFLOW_USE_SYCL
+#ifdef TF_USE_SYCLDNN
 #include "tensorflow/core/kernels/conv_ops_sycl.h"
 #endif  // TENSORFLOW_USE_SYCL
```

Comment:
// TF_USE_SYCLDNN

Reply:
I think you're looking at outdated commits. This was changed and became `#if defined(TF_USE_SYCLDNN) && defined(TENSORFLOW_USE_SYCL)` (in the same PR). Not sure what the comment should be in this case but I guess it's fine